Edit pitr compatible description with add index lightning feature (#10543)
pingcap/docs: br/pitr-known-issues.md

Issue: [#13304](https://github.com/tikv/tikv/issues/13304)

When there is a large transaction, the log checkpoint lag is not updated before the transaction is committed. Therefore, the checkpoint lag is increased by a period of time close to the commit time of the transaction.

> ## The log backup task fails when the cluster is using the acceleration of adding indexes feature

Reviewer: ```suggestion
## The acceleration of adding indexes feature is not compatible with PITR
```

Issue: [#38045](https://github.com/pingcap/tidb/issues/38045)

> Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) is not compatible with PITR. When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:

Reviewer: ```suggestion
Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) feature is not compatible with PITR. When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:
```

> Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) is not compatible with PITR. When using the index acceleration feature, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, which include:

Reviewer: ```suggestion
Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) feature is not compatible with PITR. When using the index acceleration feature, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, which include:
```

> - If you start a log backup task first, and then add an index. The adding index process is not accelerated even if index acceleration is enabled. But the index compatibility is not affected.

Reviewer 2: ```suggestion
- If you start a log backup task first, and then add an index. The adding index process is not accelerated even if index acceleration is enabled. But the index create will be finished in a slow way.
```
